crates/yakui-vulkan/src/vulkan_texture.rs

```diff
@@ -59,7 +59,7 @@ impl VulkanTexture {
         memory: vk::DeviceMemory,
         view: vk::ImageView,
     ) -> Self {
-        let address_mode = vk::SamplerAddressMode::REPEAT;
+        let address_mode = vk::SamplerAddressMode::CLAMP_TO_EDGE;
```

Comment:
Does the other mention of `REPEAT` in this file also need to be updated?

Reply:
Agh, yes.